manifests/init.pp

```diff
@@ -127,6 +127,10 @@
 #   (string) Python version to use in virtualenv.
 #   Defaults to 'system'
 #
+# [*virtualenv_dir*]
```

Comment:
can you convert this parameter into puppet-strings style? In the future we want/need to convert all existing params as well.